Assets/StreamingAssets/Quests/S0000501.txt

```diff
@@ -205,6 +206,10 @@ _S.09_ task:
 	remove log step 0
 	start task _stagger_
 
+_notyet_ task:
```

Comment:
What is this part for?

Reply:
_notyet_ ensures that the S.04 task cannot happen until S.00 is triggered. S.00 is the arrival of the queen's letter; S.04 is clicking on the queen for dialogue. Therefore, you cannot trigger dialogue with the queen until you get her letter.